lib/iris/coords.py

```diff
@@ -909,14 +917,25 @@ def is_compatible(self, other, ignore=None):
         return compatible
 
     @property
-    def dtype(self):
+    def points_dtype(self):
```

Comment:
Actually, yes, I've just thought of a reason. It's used [in the maths module](https://github.com/SciTools/iris/blob/dask/lib/iris/analysis/maths.py#L340) in a place where and object may be a cube, coord, numpy or scalar. There may be other places where it's used too